subsys/net/lib/config/Kconfig

```diff
@@ -5,6 +5,7 @@
 
 config NET_CONFIG_AUTO_INIT
 	bool "Init networking support automatically during device startup"
+	default n if USB_DEVICE_NETWORK_ECM
```

Comment:
I thing it should be
```
default n if USB_DEVICE_NETWORK
```
because netusb handles all tree types, ECM, EEM, and RNDIS

Reply:
done.